src/server/appserver.cpp

```diff
@@ -491,8 +491,8 @@ void AppServer::onRequestReceived(int id, RequestNum num, const QByteArray &para
                 sendUserUpdated(userInfo);
             }
 
-            resultStream << exitCode;
-            if (exitCode == ExitCodeOk) {
+            resultStream << enumClassToInt(exitCode);
```

Comment:
Just to check: the result is then translated back to an enum?

Reply:
It is, the: 
```
    QDataStream resultStream(&results, QIODevice::ReadOnly);
    resultStream >> exitCode;
```
and `>> operator` reinterpret_cast to exitCode:
```
template <typename T>
typename std::enable_if_t<std::is_enum<T>::value, QDataStream &>
operator>>(QDataStream &s, T &t)
{ return s >> reinterpret_cast<typename std::underlying_type<T>::type &>(t); }
```